cg/meta/workflow/analysis.py

```diff
@@ -314,7 +314,10 @@ def link_fastq_files_for_sample(
     def get_target_bed_from_lims(self, case_id: str) -> str:
         """Get target bed filename from lims"""
         case_obj: models.Family = self.status_db.family(case_id)
-        target_bed_shortname: str = self.lims_api.capture_kit(case_obj.links[0].sample.internal_id)
+        sample_obj: models.Sample = case_obj.links[0].sample
+        if sample_obj.from_sample:
+            sample_obj = self.status_db.sample(internal_id=sample_obj.internal_id)
+        target_bed_shortname: str = self.lims_api.capture_kit(sample_obj.internal_id)
         if not target_bed_shortname:
```

Comment:
What does it say here?? if NOT target_base_shortname return target_bed_shortname
Does that mean that it will return None or `""`??

Reply:
If that is the case the return type should be changed